uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,30 @@
+leave-request-record-actions {
+  display: block;
+
+  .leave-request-record-actions__button {
+    display: block;
+    float: left;
+    margin-left: 0;
+
+    &:first-child {
```

Comment:
what if you have 3 buttons? there would be no margin between the 2nd and the 3rd

either you do
```scss
.leave-request-record-actions__button {
  &:not(:first-chid) {
    margin-left
  }
}
```
or you do
```scss
.leave-request-record-actions__button {
  + & {
    margin-left
  }
}

Reply:
Done.